site/src/main/java/uk/nhs/digital/common/components/SearchComponent.java

```diff
@@ -13,48 +11,59 @@
 import org.hippoecm.hst.core.parameters.ParametersInfo;
 import org.hippoecm.hst.core.request.HstRequestContext;
 import org.hippoecm.hst.util.SearchInputParsingUtils;
+import org.onehippo.cms7.essentials.components.EssentialsSearchComponent;
+import org.onehippo.cms7.essentials.components.info.EssentialsListComponentInfo;
+import org.onehippo.cms7.essentials.components.paging.Pageable;
 import uk.nhs.digital.ps.beans.Publication;
 
 import static org.hippoecm.hst.content.beans.query.builder.ConstraintBuilder.constraint;
 import static org.hippoecm.hst.content.beans.query.builder.ConstraintBuilder.or;
 
-@ParametersInfo(type = SearchComponentInfo.class)
```

Comment:
`SearchComponentInfo` appears to no longer be used - can it be deleted?

Reply:
Yes, no longer needed so will remove